part_1/src/com/chernova/library/Library.java

```diff
@@ -0,0 +1,137 @@
+package com.chernova.library;
+
+import java.util.*;
+
+public class Library {
+    public static LinkedList<Book> list = new LinkedList<>();
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_YELLOW = "\u001B[33m";
+    public static final String ANSI_RESET = "\u001B[0m";
+
+
+    //получить все книги (метод ВОЗВРАЩАЕТ список всех книг в библиотеке)
+    public static LinkedList<Book> getAllBooks() {
+        if (list.isEmpty()) {
+            System.out.println("Библиотека пуста");
```

Comment:
why do you need it, if returning your list on 18 row will give empty list? OR you think users is not so clever to understand that list without Books is "empty library"? :)